ballerina/tests/asb_sender_receiver_negative_tests.bal

```diff
@@ -52,6 +55,46 @@ function testReceivePayloadWithIncorrectExpectedType() returns error? {
     groups: ["asb_sender_receiver_negative"],
     dependsOn: [testReceivePayloadWithIncorrectExpectedType]
 }
+function testInvalidComplete() returns error? {
+    log:printInfo("[[testInvalidComplete]");
+
+    log:printInfo("Initializing Asb sender client.");
+    MessageSender messageSender = check new (senderConfig);
+
+    log:printInfo("Initializing Asb receiver client.");
+    receiverConfig.receiveMode = RECEIVE_AND_DELETE;
+
+    MessageReceiver messageReceiver = check new (receiverConfig);
+
+    log:printInfo("Sending via Asb sender.");
+    check messageSender->send(message1);
+
+    log:printInfo("Receiving from Asb receiver client.");
+    Message|error? receivedMessage = messageReceiver->receive(serverWaitTime);
+
+    if receivedMessage is Message {
+        log:printInfo("messgae" + receivedMessage.toString());
+        Error? result = messageReceiver->complete(receivedMessage);
+        test:assertTrue(result is error, msg = "Unexpected Complete for Messages in Receive and Delete Mode");
+        test:assertTrue(regex:matches((<Error>result).message(),invalidCompleteError), msg = "Invalid Complete for " +
+        " Messages in Receive and Delete Mode");
+    } else if receivedMessage is () {
+        test:assertFail("No message in the queue.");
+    } else {
+        test:assertFail("Receiving message via Asb receiver connection failed.");
+    }
+
+    log:printInfo("Closing Asb sender client.");
+    check messageSender->close();
+
+    log:printInfo("Closing Asb receiver client.");
+    check messageSender->close();
```

Comment:
```suggestion
    check messageReceiver->close();
```

Reply:
Sure. Addressed in [cae2e53](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/196/commits/cae2e534ab7ec039301f791cdbade2f00a2f609d).